libs/utils/src/votes.ts

```diff
@@ -84,3 +88,28 @@ export const buildVoteFromCvr = ({
 
   return vote
 }
+
+/**
+ * Gets all the vote options a voter can make for a given yes/no contest.
+ * @returns ContestVoteOption[] ex. ['yes', 'no']
+ */
+export function getContestVoteOptionsForYesNoContest(
+  contest: YesNoContest
+): readonly YesNoVoteID[] {
+  assert(contest.type === 'yesno')
+  return ['yes', 'no']
+}
+
+/**
+ * Gets all the vote options a voter can make for a given contest. If write ins are allowed a single write in candidate ID is included.
```

Comment:
I know, now I'm just being picky.
```suggestion
 * Gets all the vote options a voter can make for a given contest. If write-ins are allowed a single write-in candidate ID is included.
```